### ISSUE TITLE
Python 3.7 clickhouse_connect import error importlib.metadata

### ISSUE THREAD

**Issue by morooshka** [ClickHouse/clickhouse-connect#245]
### Describe the bug

Python `3.7`,  Module `clickhouse_connect`
Since version 0.6.9 there is an import error because `importlib.metadata` in python3.7 is `importlib_metadata`

### Steps to reproduce

Just import:
~~~
> python3 -c "import clickhouse_connect"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/alexander/.local/lib/python3.7/site-packages/clickhouse_connect/__init__.py", line 2, in <module>
    from clickhouse_connect.entry_points import validate_entrypoints
  File "/home/alexander/.local/lib/python3.7/site-packages/clickhouse_connect/entry_points.py", line 6, in <module>
    from importlib.metadata import PackageNotFoundError, distribution
ModuleNotFoundError: No module named 'importlib.metadata'
~~~

### Expected behaviour

No error on import

### Code example

```python
import clickhouse_connect
```

**Comment by genzgd** [comment]
Sorry about that, this should be fixed in 0.6.14.  Python 3.7 is EOL and no longer included in our CI tests, which is why I missed this.  I strongly recommend upgrading. :)

**Comment by morooshka** [comment]
Hi @genzgd!
Thank you for reply.
Yes, upgrading is needed, but this is PROD Airflow and it is not quick. You know how it is =)
